docs/Prelude.md

```diff
@@ -983,4 +983,20 @@ instance showArray :: (Show a) => Show (Array a)
 instance showOrdering :: Show Ordering
 ```
 
+#### `NaturalTransformation`
+
+``` purescript
+type NaturalTransformation f g = forall a. f a -> g a
+```
+
+A natural transformation
+
+#### `Natural`
+
+``` purescript
+type Natural f g = NaturalTransformation f g
```

Comment:
My other complaint about `Natural` is that we might want to reserve that name in something like `Prelude` for a natural number type.

Also, let me play devil's advocate for a second and say - since we can define a synonym (or not!) anywhere, what's the advantage of adding this to Prelude?